### ISSUE TITLE
5th ed. missing word

### ISSUE THREAD

**Issue by literallyalice** [rbwhitaker/CSharpPlayersGuideEarlyAccess#713]
![image](https://github.com/rbwhitaker/CSharpPlayersGuideEarlyAccess/assets/1873012/dea0f465-c0fd-45ee-a916-f9a9faa54e4e)

> Give your program a traditional Program and Main method instead of top-level statements.

should be

> Give your program a traditional Program class and Main method instead of top-level statements.

**Comment by rbwhitaker** [comment]
I agree, this is better phrasing. I suspect people know the intent, but your version is definitely an improvement. I've added an errata item for anybody else who might stumble upon this.
